.github/workflows/docs.yml

```diff
@@ -15,7 +15,7 @@ on:
       - '*'
 
 env:
-  USE_CACHE: ${{ (github.event_name == 'workflow_dispatch' && github.event.inputs.cache == 'true') || (github.event_name == 'pull_request') }}
+  USE_CACHE: ${{ (github.event_name == 'workflow_dispatch' && github.event.inputs.cache == 'true') || (github.event_name == 'pull_request') || (github.event_name == 'push') }}
```

Comment:
This would use the cache on releases, not building from scratch. I think we want to build from scratch on releases

Reply:
> This would use the cache on releases, not building from scratch

We bump the version on release, and that triggers a cache miss forcing a rebuild. We'll do that both when creating the main branch (e.g. MAJOR.MINOR.dev0), and bumping the main branch to MAJOR.MINOR.PATCH)

Reply:
Ah, I forgot that we use the version file's hash for the cache key. Then this should be good by me!